spec/views/candidates/home/index.html.erb_spec.rb

```diff
@@ -36,34 +36,4 @@
     it { is_expected.to have_css('a.govuk-button', text: 'Start now') }
     it { is_expected.to have_css('#candidate-alert-notification') }
   end
-
-  describe 'Used the service before?' do
-    let(:heading_text) { 'Used the service before?' }
-
-    context 'when disabled' do
-      before do
-        allow(Feature).to receive(:enabled?).with(:candidates_dashboard) { false }
-      end
-
-      before { render }
-
-      specify 'should not allow the candidate to sign in' do
-        expect(rendered).not_to have_css('h2', text: heading_text)
-        expect(rendered).not_to have_link('Sign in', href: candidates_dashboard_path)
-      end
-    end
-
-    context 'when enabled' do
```

Comment:
Do we not want to keep these tests?

Reply:
I thought these would be fine to go as they are just testing the feature flag.

We don't have tests for the other links on the homepage